cartography/intel/gcp/compute.py

```diff
@@ -188,4 +493,10 @@ def sync(session, compute, project_id, gcp_update_tag, common_job_parameters):
     :return: Nothing
     """
     logger.info("Syncing Compute objects for project %s.", project_id)
-    sync_gcp_instances(session, compute, project_id, gcp_update_tag, common_job_parameters)
+    zones = get_zones_in_project(project_id, compute)
+    # Only pull additional assets for this project if the Compute API is enabled
+    if zones:
+        regions = _zones_to_regions(zones)
+        sync_gcp_vpcs(session, compute, project_id, gcp_update_tag, common_job_parameters)
```

Comment:
Is it possible to have VPCs without zones?

Reply:
Yes.  A VPC is a [global resource](https://cloud.google.com/compute/docs/regions-zones/global-regional-zonal-resources#globalresources) that doesn't depend on a zone, and the URI for a VPC doesn't have a zone in it.

This check is more to ensure that the Compute API is enabled for the given project.  This code currently makes the assumption that if there are no zones returned, then the Compute API is disabled.

That said, good catch - I actually don't know if it is possible for the Compute API to be enabled **and** 0 zones are returned.  I'll make an adjustment to account for this.

Reply:
From chatting in person, this seems more explicit to me:
```
if zones is None:
    return  # compute is not enabled
else:
    regions = _zones_to_regions(zones)
    ...
```